scripts/travis/configure-make.sh

```diff
@@ -0,0 +1,35 @@
+# raw Makefile configuration
+
+LINE () {
+  echo "$@" >> Makefile.config
+}
+
+cp Makefile.config.example Makefile.config
+
+LINE "BLAS := open"
+LINE "WITH_PYTHON_LAYER := 1"
+
+if $WITH_PYTHON3 ; then
+  # XXX doesn't work
```

Comment:
Oh, I read this as the author of the TODO, and not the assignee, so `TODO(lukeyeager)` is fine. Don't worry, it's not a promise to do it yourself haha. Although versioning records the line author it's nice to have a quick reference for who thought of the TODO in case there are later questions or you want to search for commits by the same author for context.